### PR TITLE
WIP: Create hero header component.

### PR DESCRIPTION
**Pull request by nirgn975**
#19

### CODE REVIEW

File: client/src/app/managing-body/managing-body.service.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    label: string;
+}
+
+@Injectable()
+export class ManagingBodyService  {
+
+  constructor(private http: Http) {}
+
+  getManagingBody() {
+    return this._http.get('http://localhost:8000//api/managing_bodies/?format=json')
```
Comment (ordavidil) [inline client/src/app/managing-body/managing-body.service.ts]: @nirgn975 
Create a _Config_ which will include the backend url.

File: client/src/app/hero-header/hero-header.component.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+op-hero-header {
+  background-color: ##f0f0f0;
+}
```
Comment (ordavidil) [inline client/src/app/hero-header/hero-header.component.scss]: new line

File: client/src/app/app.ts [hunk 3]
```diff
@@ -13,10 +14,13 @@ import { About } from './about/about.component';
 })
 
 @RouteConfig([
-  { path: '/home',       component: Home,        name: 'Home', useAsDefault: true },
-  { path: '/about',      component: About,       name: 'About' },
+  { path: '/home',                component: HomeComponent,          name: 'Home', useAsDefault: true },
```
Comment (ordavidil) [inline client/src/app/app.ts]: break the object to separate lines.

File: client/src/app/app.html [hunk 4]
```diff
@@ -1,21 +1,18 @@
...
-	<a [routerLink]=" ['/Home'] ">
-    Home
-  </a>
-  |
-	<a [routerLink]=" ['/About'] ">
-    About
-  </a>
+	<ul>
+		<li>
+			<a [routerLink]=" ['Home'] ">Home</a>
```
Comment (ordavidil) [inline client/src/app/app.html]: Set your IDE configuration so the format of the indentation will be 2 spaces.

Comment (ordavidil) [inline client/src/app/app.html]: And fix indentation :)

File: client/src/app/managing-body/managing-body.service.ts (hunk 1 above)
Comment (nirgn975) [inline client/src/app/managing-body/managing-body.service.ts]: Oh for sure, I'm going through john papa style guide, it's still WIP..

File: client/src/app/hero-header/hero-header.component.scss (hunk 2 above)
Comment (nirgn975) [inline client/src/app/hero-header/hero-header.component.scss]: Yeah I forget about that, but I'm actually intend to delete the SCSS file, it's just for testing, still WIP..

File: client/src/app/app.ts (hunk 3 above)
Comment (nirgn975) [inline client/src/app/app.ts]: I'm not sure about that, I don't see anyone break the routeing to separate lines, not in Angular style guide, or John Papa one. But I'll check it out before submitting.
And I read now about child routeing, maybe it's better that way, don't know yet..

File: client/src/app/app.html (hunk 4 above)
Comment (nirgn975) [inline client/src/app/app.html]: Yeah, I did it from Pycharm, my bad (:

**Pull request closed without merging**